src/primitives/redpallas.rs

```diff
@@ -23,7 +23,7 @@ pub type Binding = reddsa::orchard::Binding;
 impl SigType for Binding {}
 
 /// A RedPallas signing key.
-#[derive(Clone, Debug)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
str4d ([Link](https://github.com/zcash/orchard/pull/471#discussion_r2549938764)): We shouldn't allow signing keys to be arbitrarily copied around the stack.
```suggestion
#[derive(Clone, Debug)]
```